faq/running.inc

```diff
@@ -1663,6 +1663,19 @@ For more information refer to
 libfabric.org</a> web site.
 ";
 
+$q[] = "How can Open MPI communicate with Intel Omni-Path Architecture (OPA)
+based devices?";
+
+$anchor[] = "opa-support";
+
+$opa = "<a href\"https://www.intel.com/content/www/us/en/support/products/92003/network-and-i-o/fabric-products/intel-omnipath-host-fabric-interface-products.html\">";
+
+$a[] = "Currently, Open MPI supports PSM2 MTL and OFI MTL components which can
```

Comment:
minor comment:  ...  OFI MTL (using PSM2 OFI provider) ...

Reply:
Fixed in new commit